components/PostCard.tsx

```diff
@@ -1,13 +1,43 @@
+import {
+  isPostADraft,
+  isPostInTheFuture,
+} from "helpers/checkOfDraftOrFuturePost";
 import { PostDocumentWithoutContent } from "interfaces";
 import Image from "next/image";
 import Link from "next/link";
 import { useAppDispatch } from "redux/typesHooks";
 
 import { setTags } from "../redux/slices/selectedTags";
+
 interface Props {
   post: PostDocumentWithoutContent;
 }
 
+export const DraftPostMark = () => (
+  <div className="draft-post">
+    <div className="triangle-draft triangle triangle-item">
+      <span className="triangle-text-draft triangle-text">draft</span>
+    </div>
+    <div className="small-triangles-draft">
+      <div className="small-triangles-item-draft-first triangle small-triangles-item-draft small-triangles-item"></div>
+      <div className="small-triangles-item-draft-second triangle small-triangles-item-draft small-triangles-item"></div>
```

Comment:
Remove those two divs, and just use CSS pseudo-elements
They are designed for such "decorated" things

Something like:
https://codepen.io/chriscoyier/pen/OmORqR